core/todo.txt

```diff
@@ -73,8 +73,16 @@ X play with improvements to full screen here
 
 
 showstoppers
+<<<<<<< HEAD
```

Comment:
And please take care not to mess with the todo files... There should never be any changes to those in a PR.